In [1]:
import numpy as np
import pandas as pd

In [3]:
TFs = pd.read_csv('https://raw.githubusercontent.com/yjgeno/Xct/dev2/DB/41587_2020_742_MOESM3_ESM.csv', header=None, names=['TF_symbol'])
TFs

TF_symbol
0         ADNP
1        AEBP2
2         AFF1
3         AFF4
4       AHCTF1
...        ...
1559     ZUFSP
1560      ZXDA
1561      ZXDB
1562      ZXDC
1563      ZZZ3

[1564 rows x 1 columns]

Refer to TFome, a comprehensive library containing 1,564 TF genes: <br>
https://www.nature.com/articles/s41587-020-0742-6